.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
+**Important**: This is a public repository. Anyone in the world can see what's posted here. If you are posting screenshots or log files, please **carefully examine them for** the presence of any kind of **protected health information** (PHI). Images or logs containing PHI _must_ be posted in fully-redacted form, with no visible PHI.
```

Comment:
We still get some people submitting issues without remembering to delete this statement. I _think_ that wrapping it in HTML comment tags stops GH from including it in the final issue while still showing it during creation. There is some prior art in cht-core: https://raw.githubusercontent.com/medic/cht-core/master/.github/PULL_REQUEST_TEMPLATE.md

If you agree, use this on all the sections below too!

```suggestion
<!--
**Important**: This is a public repository. Anyone in the world can see what's posted here. If you are posting screenshots or log files, please **carefully examine them for** the presence of any kind of **protected health information** (PHI). Images or logs containing PHI _must_ be posted in fully-redacted form, with no visible PHI.
-->
```

Reply:
Ohhh love this! Thanks for directing me to the examples!